drivers/ethernet/eth_e1000.c

```diff
@@ -134,22 +134,19 @@ static void e1000_isr(struct device *device)
 
 int e1000_probe(struct device *device)
 {
-	struct e1000_dev *dev = device->driver_data;
-
-	pci_bus_scan_init();
-
-	if (pci_bus_scan(&dev->pci)) {
-
-		pci_enable_regs(&dev->pci);
+	const pcie_bdf_t bdf = PCIE_BDF(0, 3, 0);
+	const u32_t flags = PCIE_CONF_CMDSTAT_MEM | PCIE_CONF_CMDSTAT_MASTER;
 
-		pci_enable_bus_master(&dev->pci);
-
-		pci_show(&dev->pci);
+	struct e1000_dev *dev = device->driver_data;
+	int retval = -ENODEV;
 
-		return 0;
+	if (pcie_probe(PCIE_BDF(0, 3, 0), PCIE_ID(0x8086, 0x100e))) {
```

Comment:
Why is literal `PCIE_BDF(0, 3, 0)` used here again, while it previously as defined as `bdf` constant?

Reply:
Mere oversight.  Fixed.